index.bs

```diff
@@ -8440,6 +8440,48 @@ corresponding to [=interface members=].
 </div>
 
 
+<h4 id="InNamespace" extended-attribute lt="InNamespace">[InNamespace]</h4>
+
+If the [{{InNamespace}}] [=extended attribute=] appears on an [=interface=], it indicates that
+the [=interface object=] for this interface will not be created as a property of the global
+object, but rather the namespace which is the argument to the extended attribute.
```

Comment:
"but rather as a property of" is clearer, I think

"namespace identified by the argument", I think

`[=namespace=]`

Reply:
Done